training/src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,87 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity (name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
```

Comment:
In the most scenarios, it is better to use the merge cascade. Thus, you avoid to delete data and relationships
```suggestion
    @OneToMany(mappedBy = "user", cascade = CascadeType.MERGE)
```